rspack.embedding-sdk-bundle.config.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-env node */
+/* eslint-disable import/no-commonjs */
+/* eslint-disable import/order */
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+const rspack = require("@rspack/core");
+const BundleAnalyzerPlugin =
+  require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+
+const mainConfig = require("./rspack.main.config");
+const { resolve } = require("path");
+const fs = require("fs");
+const path = require("path");
+
+const {
+  TypescriptConvertErrorsToWarnings,
+} = require("./frontend/build/embedding-sdk/rspack/typescript-convert-errors-to-warnings");
+const {
+  LICENSE_TEXT,
+  IS_DEV_MODE,
+} = require("./frontend/build/shared/constants");
+const {
+  OPTIMIZATION_CONFIG,
+} = require("./frontend/build/embedding-sdk/rspack/shared");
+const { BABEL_CONFIG } = require("./frontend/build/shared/rspack/babel-config");
+const { CSS_CONFIG } = require("./frontend/build/shared/rspack/css-config");
+const {
+  getBannerOptions,
+} = require("./frontend/build/shared/rspack/get-banner-options");
+
+const SDK_SRC_PATH = __dirname + "/enterprise/frontend/src/embedding-sdk";
+const BUILD_PATH = __dirname + "/resources/embedding-sdk";
+
+const ENTERPRISE_SRC_PATH =
+  __dirname + "/enterprise/frontend/src/metabase-enterprise";
+
+const skipDTS = process.env.SKIP_DTS === "true";
+
+const isDevMode = IS_DEV_MODE;
+
+const sdkPackageTemplateJson = fs.readFileSync(
+  path.resolve(
+    path.join(
+      __dirname,
+      "enterprise/frontend/src/embedding-sdk/package.template.json",
+    ),
+  ),
+  "utf-8",
+);
+const sdkPackageTemplateJsonContent = JSON.parse(sdkPackageTemplateJson);
+const EMBEDDING_SDK_VERSION = sdkPackageTemplateJsonContent.version;
+
+const shouldAnalyzeBundles = process.env.SHOULD_ANALYZE_BUNDLES === "true";
+
+const config = {
+  ...mainConfig,
+
+  context: SDK_SRC_PATH,
+
+  entry: "./index.ts",
+
+  output: {
+    path: BUILD_PATH + "/dist",
+    publicPath: "",
+    filename: "[name].bundle.js",
+    library: {
+      type: "commonjs2",
+    },
+  },
+
+  devtool: isDevMode ? mainConfig.devtool : false,
```

Comment:
nit: wouldn't this already be made `false` by the `mainConfig.devtool` if we use the same env for isDevMode there?